src/main/java/chess/dao/ChessBoardDAO.java

```diff
@@ -0,0 +1,94 @@
+package chess.dao;
+
+import chess.domain.dto.MoveRequestDTO;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChessBoardDAO {
+
+    private Connection getConnection() {
+        Connection con = null;
+        String server = "localhost:13306"; // MySQL 서버 주소
+        String database = "chess_game"; // MySQL DATABASE 이름
+        String option = "?useSSL=false&serverTimezone=UTC";
+        String userName = "root"; //  MySQL 서버 아이디
+        String password = "root"; // MySQL 서버 비밀번호
+
+        // 드라이버 로딩
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            System.err.println(" !! JDBC Driver load 오류: " + e.getMessage());
+            e.printStackTrace();
+        }
+
+        // 드라이버 연결
+        try {
+            con = DriverManager
+                .getConnection("jdbc:mysql://" + server + "/" + database + option, userName,
+                    password);
+            System.out.println("정상적으로 연결되었습니다.");
+        } catch (SQLException e) {
+            System.err.println("연결 오류:" + e.getMessage());
+            e.printStackTrace();
+        }
+
+        return con;
+    }
+
+    public void closeConnection(Connection con) {
+        try {
+            if (con != null) {
+                con.close();
+            }
+        } catch (SQLException e) {
+            System.err.println("con 오류:" + e.getMessage());
+        }
+    }
+
+
+    public void addLog(String source, String target) throws SQLException {
+        String query = "INSERT INTO log VALUES (?, ?)";
+        Connection connection = getConnection();
```

Comment:
```suggestion
        try (Connection connection = getConnection()) {
```

https://github.com/woowacourse/java-chess/pull/233#discussion_r606637157에 대해 답변드리자면..!
try-with-resources를 사용한다면 예외 상황에서도 자원을 회수할 수 있습니다 :)

Reply:
변경했습니다! 자바 9 이후 도입된 문법으로 사용하려고 했는데 자바 8로 빌드했었더라구요
그냥 자바 7에 도입된 문법대로 사용했습니다!